server/src/events.mjs

```diff
@@ -48,22 +52,31 @@ function logFatalErr() {
 // name (e.g., lifecycle.onInactive) to message id (e.g., 17)
 const eventListenerMap = {};
 
-// Associate this message ID with this method so if/when events are sent, we know which message ID to use
-function registerEventListener(userId, oMsg, ws) {
+/**
+ * Associate this message ID with this method so if/when events are sent, we know which message ID to use
+ * @param {string} userId - The user ID associated with the event listener
+ * @param {Object} metadata - The metadata object containing information about the event listener registration
+ * @param {WebSocket} ws - The WebSocket object associated with the event listener
+ * @returns {void}
+*/
+function registerEventListener(userId, metadata, ws) {
+  const { registration, method } = metadata;
+
   if (!eventListenerMap[userId]) {
     eventListenerMap[userId] = {};
   }
 
-  if (!eventListenerMap[userId][oMsg.method]) {
-    eventListenerMap[userId][oMsg.method] = { id: oMsg.id, wsArr: [] };
+  if (!eventListenerMap[userId][method]) {
+    // id: registration.id will be removed in part 2
```

Comment:
Wondering if this represents something left-over / to do since this is the PR for part 2, right?

Reply:
Nice catch, forgot to clean up!